app/presenters/contact_gone_presenter.rb

```diff
@@ -15,6 +17,12 @@ def present
       routes: [
         { path: contact.link, type: "exact" }
       ],
+      content_id: content_id,
```

Comment:
I would recommend also adding a `base_path` while you're at it.  It's not a required field yet but I suspect it might be in future.